testing/rclone/APKBUILD

```diff
@@ -1,6 +1,7 @@
+# Contributor: Sandro Jäckel <sandro.jaeckel@posteo.de>
```

Comment:
Usually contributor fields are added for more significant changes :sweat_smile:

Reply:
``Usually``
I also want to bump it to 1.50.2